tresgdl:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  shoulder_pan_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_pan
    pid: {p: 100.0, i: 0.01, d: 10.0}
  shoulder_tilt_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_tilt
    pid: {p: 100.0, i: 0.01, d: 10.0}
  elbow_position_controller:
    type: effort_controllers/JointPositionController
    joint: elbow
    pid: {p: 100.0, i: 0.01, d: 10.0}